usr/src/man/man1m/getent.1m

```diff
@@ -76,8 +77,9 @@ uses the appropriate library routines described in
 .Xr getauthattr 3SECDB ,
 .Xr getexecattr 3SECDB ,
 .Xr getprofattr 3SECDB ,
+.Xr getuserattr 3SECDB ,
 and
-.Xr getuserattr 3SECDB
+.Xr getnetgrent 3C
```

Comment:
Sorry I didn't pick up on this before, but these are usually kept sorted by section then name - getnetgrent should be up with the other `3C` entries.